package/recalbox-system/Config.in

```diff
@@ -2,3 +2,25 @@ config BR2_PACKAGE_RECALBOX_SYSTEM
     bool "Recalbox System"
     help
       Install the recalbox system files
+
+choice
```

Comment:
This should be directly in the def-config. No choices for the end user as one could enable xu4 here, but compile in armv6.